pkg/constant/pattern.go

```diff
@@ -30,7 +31,8 @@ func GenerateClusterComponentName(clusterName, compName string) string {
 
 // GenerateAccountSecretName generates the secret name of system accounts.
 func GenerateAccountSecretName(clusterName, compName, name string) string {
-	return fmt.Sprintf("%s-%s-account-%s", clusterName, compName, name)
+	replacedName := strings.ReplaceAll(name, "_", "-")
+	return fmt.Sprintf("%s-%s-account-%s", clusterName, compName, replacedName)
```

Comment:
I think it's a common preprocessing step, 

code like 
```
strings.ReplaceAll(name, "_", "-")
```

is better to be extract to a lib code, and well-documented.

Reply:
This is just a suggestion, can be done in the following pr.

Reply:
> This is just a suggestion, can be done in the following pr.

yes, it will be considered later. Currently, only the systemAccount cannot restrict the naming rules due to specific reasons related to the engine. Almost all other objects are restricted by validation rules. If needed in the future, we can handle it uniformly.